### DISCUSSION TITLE
Project manage

### DISCUSSION THREAD

**Discussion by syinwu** [corazawaf/coraza#148] [General]
It is recommended to create an organization to manage the project.

**Comment by jptosso** [comment]
Hey! I agree, but I haven't done it yet because I don't know if Coraza will become part of a bigger foundation, or it will become Coraza foundation. It's something that requires a lot of analysis, and you are welcome to participate in the decision. But Coraza foundation sounds really cool and I think, as a community, we can make it work

I created this organization a few years ago for Coraza: https://github.com/corazawaf

**Comment by jcchavezs** [comment]
Shall we close this?

**Comment by jptosso** [comment]
Well now we are part of OWASP so I guess so
